src/core/managers/internal/ToastManager/helpers/tests/createToastsFromTransactions.test.ts

```diff
@@ -0,0 +1,211 @@
+import { testAddress } from '__mocks__/accountConfig';
```

Comment:
why not have testAccount also here?

Reply:
No need for now, as we do not use account mocks